src/Service/Database/Migrations/Ajax/Jobs/SummaryDataMigrator.php

```
@@ -0,0 +1,58 @@
+     *
+     * @var int
+     */
+    protected $offset = 0;
+
+    /**
+     * Retrieves the total count of data to be migrated.
+     *
+     * @param bool $needCaching Whether to load/save from cache. Defaults to true.
+     * @return void
+     */
+    protected function getTotal($needCaching = true)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$needCaching'. (undefined)

(UnusedFormalParameter)